containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -901,33 +924,34 @@ class UnorderedMap {
   bool modified() const { return get_flag(modified_idx); }
 
   void set_flag(int flag) const {
-    using raw_deep_copy =
-        Kokkos::Impl::DeepCopy<typename device_type::memory_space,
-                               Kokkos::HostSpace>;
-    const int true_ = true;
-    raw_deep_copy(m_scalars.data() + flag, &true_, sizeof(int));
+    Kokkos::parallel_for(
+        "set flag",
```

Comment:
Here and below prefix the label  with `Kokkos::`

Reply:
Done.